src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -43,6 +43,11 @@ import scala.runtime.Statics
   *  // 2
   *  // 3
   *  }}}
+  * 
+  *  A `LazyList`, like the one in the example above, may be infinite in length.
+  *  Aggregate methods, such as `count`, `sum`, `max` or `min` on such infinite length 
+  *  sequences will not terminate. Filtered infinite lazy lists also can't be fully
+  *  iterated and aggregates don't terminate on such lists either.
```

Comment:
I actually prefer the original wording @martijnhoekstra had here, sorry @som-snytt

Reply:
That doesn't surprise me.